src/cli.ts

```diff
@@ -197,7 +197,7 @@ prog.command('export [dest]')
 	.option('--host', 'Host header to use when crawling site')
 	.option('--concurrent', 'Concurrent requests', 8)
 	.option('--timeout', 'Milliseconds to wait for a page (--no-timeout to disable)', 5000)
-	.option('--legacy', 'Create separate legacy build')
+	.option('--legacy', 'Creates a separate legacy build for older browsers')
```

Comment:
This still repeats the option's name to a degree, I'd opt for something even simpler: 

`"Creates an additional build compatible with older browsers"`

Reply:
As I understand this additional build is only served to older browsers, I have used that language

Reply:
You used the word `legacy` again, isn't that the term that was confusing in the first place?

Reply:
I wanted to know what was meant by legacy and how that would affect my app if I removed it. The key information is that it's an `additional` build and (potentially) served only to older browsers